plugins/tracker-resources/src/issues.ts

```diff
@@ -64,7 +65,9 @@ export async function issueLinkProvider (doc: Doc): Promise<string> {
 
 export function generateIssueShortLink (issueId: string): string {
   const location = getCurrentResolvedLocation()
-  return `${window.location.protocol}//${window.location.host}/${workbenchId}/${location.path[1]}/${trackerId}/${issueId}`
```

Comment:
It should be more places like this one, so please check for window.location usage.